azure/scope/cluster_test.go

```diff
@@ -160,6 +160,7 @@ func TestAPIServerHost(t *testing.T) {
 	}
 
 	for _, tc := range tests {
+		tc := tc
```

Comment:
The `gosec` linter found several instances of `G601: Implicit memory aliasing in for loop` in tests. (This is no longer an issue in Go 1.22, but for now it's still a potential bug.)